shared/lib/names.js

```diff
@@ -20,6 +20,25 @@ function kebabCaseToPascalCase(value) {
   )
 }
 
+/**
+ * Convert a kebab-cased string to a camelCased one
+ *
+ * @param {string} value - Input kebab-cased string
+ * @returns {string} Output camelCased string
+ */
+function kebabCaseToCamelCase(value) {
+  return (
+    value
+      .toLowerCase()
+      .split('-')
+      // capitalize each 'word'
+      .map((word, index) => {
+        return index > 0 ? word.charAt(0).toUpperCase() + word.slice(1) : word
+      })
+      .join('')
+  )
```

Comment:
Do you mind me saying I liked your original idea?

Where you just lowercased the first **PascalCase** letter back into **camelCase**

```suggestion
  return kebabCaseToPascalCase(value).replace(/^./, (str) =>
    str.toLowerCase()
  )
```